[19] Logout fixes for emulated interface

ShantKhatri opened the pull request:
fixes #19 
- Added conditions to maintain the flow of logout on the emulated interface.

ShantKhatri committed:
Added emulation for chrome and edge

ShantKhatri committed:
Reinitialization methode fixes

ShantKhatri committed:
Browser name fixes

ShantKhatri committed:
logout flow fixes for iPad mini emulation

infa-rvarahag reviewed (changes requested):
No sysouts , use  log stmts please

ShantKhatri committed:
Changed emulation interface from iPad mini to pixel 4a

Changed emulation interface from iPad mini to pixel 4a

This reverts commit a5364776cbd22154922dee790ce3a99b54dd7449.

Changed emulation interface from iPad mini to pixel 4a
